client/my-sites/stats/activity-log-item/index.jsx

```diff
@@ -196,7 +197,7 @@ class ActivityLogItem extends Component {
 			<div className={ classes } >
 				<div className="activity-log-item__type">
 					{ this.renderTime() }
-					<ActivityIcon { ...pick( log, [ 'activityName', 'activityIcon' ] ) } />
+					<ActivityIcon { ...pick( log, [ 'activityIcon', 'activityName', 'activityStatus' ] ) } />
```

Comment:
Is `ActivityIcon` still using `activityName`? (Cf https://github.com/Automattic/wp-calypso/pull/18179/files#r140901125)

Reply:
If it doesn't, the destructured `pick` seems a bit overkill for two props. Maybe destructure them from `log` further up and pass `activityIcon` and `activityStatus` explicitly here?

Reply:
Definitely!